api/modules/checklist/model.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Checklist(models.Model):
+    user = models.ForeignKey(User, related_name='checklist', on_delete=models.CASCADE)
+    last_updated = models.DateTimeField(auto_now=True)
+
+
+class ChecklistItem(models.Model):
```

Comment:
Since this is a "check"list, there needs to be a status. Also you'd need to add two more fields: `created_at` and `last_updated_at`